Bevy 0.17 release notes editorial pass (#21108)
bevyengine/bevy: release-content/release-notes/raymarched-atmosphere-space-views.md

(Insert screenshot of space shot including volumetric shadows)

Bevy's atmosphere now supports a raymarched rendering path that unlocks accurate views from above the atmosphere. This means **Bevy 0.17** now has two atmosphere rendering modes to choose from:

- [`AtmosphereMode::Raymarched`]

>   - Ideal for cinematic shots, planets seen from space, and scenes that need sharp shadows through the volume of the atmosphere (such as flight simulators)
>   - More accurate lighting, but slower
>   - Sharper shadows through the atmosphere

Reviewer: sharper shadows is mentioned twice
```suggestion
  - Ideal for cinematic shots, planets seen from space, and flight simulator type scenes
  - More accurate lighting, but slower
  - Sharper shadows through the atmosphere
```

- [`AtmosphereMode::LookupTexture`]
  - This is the default
  - Great for ground level and broad outdoor scenes
  - Less accurate lighting (especially at long distances), but faster
  - Softer shadows through the atmosphere

To use it, add an [`Atmosphere`] component to your [`Camera`] and set the rendering method on the camera’s [`AtmosphereSettings`]:

```rust
commands.spawn((
    Camera3d::default(),
    Atmosphere::default(),
    AtmosphereSettings { 
      rendering_method: AtmosphereMode::Raymarched, 
      ..Default::default() 
    }
));
```

> You can also adjust the `AtmosphereSettings::sky_max_samples` to configure the maximum number of steps to take when raymarching the atmosphere. Lower numbers are faster and less accurate. Higher numbers are slowly and more accurate.

Reviewer: ```suggestion
You can also adjust the `AtmosphereSettings::sky_max_samples` to configure the maximum number of steps to take when raymarching the atmosphere. Lower numbers are faster and less accurate. Higher numbers are slower and more accurate.
```
